pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -25,6 +46,13 @@ def __init__(self, *args, **kwargs):
         self.failed_only = kwargs.get('failed_only', False)
         self.fail_fast = kwargs.get('fail_fast', False)
         self.run_under_coverage = kwargs.get('with_coverage', True)
+        django_version = kwargs.get('django_version', None)
+        if django_version is None:
+            self.django_toxenv = None
+        elif django_version != '1.8':
+            self.django_toxenv = 'py27-django111'
+        else:
+            self.django_toxenv = 'py27-django18'
```

Comment:
nit: if you think it improves clarity, can you swap/invert the conditions:
```python
        elif django_version == '1.8':
            self.django_toxenv = 'py27-django18'
        else:
            self.django_toxenv = 'py27-django111'
```

Reply:
My intent was to default to `1.8` if anything unparseable was specified. I'll try to make this more clear.